src/components/Chatbot/Chatbot.jsx

```diff
@@ -91,40 +92,42 @@ const steps = [
             { value: 47, label: 'A beautiful garden with beautiful flowers and plants', trigger: 'end' },
             { value: 48, label: 'Showcase all four seasons on one piece of paper', trigger: 'end' },
             { value: 49, label: 'Draw how innocent and happy children are', trigger: 'end' },
-            { value: 50, label: 'Various breeds of dogs in one picture', trigger: 'end' }
+            { value: 50, label: 'Various breeds of dogs in one picture', trigger: 'end' },
         ],
     },
     {
         id: 'end',
         component: (
             <div>
-                Great Choice! Good luck with your drawing!
-                <button onClick={() => window.location.reload()}>End Chat</button>
+                Great Choice! Good luck with your drawing!<br />
+                <button onClick={() => window.location.reload()} 
+                    style={{ color: 'blue', marginTop: '10px' }}>End Chat</button>
```

Comment:
You need to use [TailwindCSS](https://tailwindcss.com/docs/installation) instead of vanilla CSS.

```suggestion
<button 
  onClick={() => window.location.reload()} 
  className="text-blue-800 mt-3"
  >
   End Chat
</button>
```

Reply:
oh, sorry for that, used blue-500